<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorDataLogger</class>
 <widget class="QMainWindow" name="MotorDataLogger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MotorDataLogger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="heading">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Data Logger</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnected">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>530</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Device not connected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="portList"/>
       </item>
       <item>
        <widget class="QComboBox" name="baudList"/>
       </item>
       <item>
        <widget class="QPushButton" name="connectPushBtn">
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectPushBtn">
         <property name="text">
          <string>disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start Logging Motor Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Stop Logging Motor Data</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
